package.json

```diff
@@ -136,6 +136,7 @@
       "no-console": 1,
       "no-use-before-define": 0,
       "prefer-template": 2,
+      "react/forbid-prop-types": 0,
```

Comment:
After having a look at the components that use 'forbidden prop types' (any, array and object) instead of something more descriptive (like 'shape'), they look like valid and open uses of those prop types.

I've decided to ignore this linting rule as the rule is `mainly for documentation` (https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-prop-types.md#when-not-to-use)